marklogic-data-hub/src/main/resources/ml-modules/root/data-hub/5/impl/hub-utils.sjs

```diff
@@ -146,6 +146,14 @@ function writeDocument(docUri, content, permissions, collections, database) {
   }));
 }
 
+function getObjectValues(object){
```

Comment:
Can you include a comment on this stating that the reason for its existence is that we have to support ML 9, and ML 9 does not support Object.values?

Reply:
Great, thanks - this is the kind of stuff that's easy to forget later, such that a future developer says "Well we don't need this, we can just use Object.values", not knowing that it won't work on ML 9.